manifests/pod.yaml

```diff
@@ -24,6 +24,12 @@ spec:
          value: /.aws-fcos-builds-bot-config/config
        - name: AWS_FCOS_KOLA_BOT_CONFIG
          value: /.aws-fcos-kola-bot-config/config
+       - name: GCP_IMAGE_UPLOAD_CONFIG
+         value: /.gcp-image-upload-config
+       - name: GCP_KOLA_TESTS_CONFIG
+         value: /.gcp-kola-tests-config
```

Comment:
Yeah... we should really look at constraining each container run to the minimal set of credentials it needs. E.g. no reason the AWS kola test needs GCP creds mounted in.

Reply:
I assume you mean something for us to do in the future?

Reply:
Yup, definitely!